packages/blade/src/components/Tooltip/TooltipInteractiveWrapper.web.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import BaseBox from '~components/Box/BaseBox';
+import { makeBorderSize } from '~utils';
+
+const TooltipInteractiveWrapper = styled(BaseBox).attrs(() => {
+  return { tabIndex: -1 };
+})((props) => {
+  return {
+    display: 'inline-block',
+    '&:focus': {
+      borderRadius: makeBorderSize(props.theme.border.radius.medium),
+      // TODO: Replace with focus outline token
+      outline: `1px solid ${props.theme.colors.surface.background.level1.lowContrast}`,
```

Comment:
when we add an Interactive wrapper shouldn't we make the trigger element focussable?

Reply:
Actually no, i started thinking that it should be focusable. and have tabIndex={0} but then realised a non-interactive component should always be non-interactive. 

Imagine someone adds a tooltip around an image or icon, the image should not suddenly become focusable, if it does it will confuse screen reader users that why does an image getting focus? "is this clickable?". 

Instead the image should define it's presence via the alt text and the associated label (aria-labelledby) which is added by the Tooltip. 

Same goes for other non-interactive elements, imagine a non-interactive InfoIcon besides an Input.
In this case: 

1. Any critical information should not be hidden under a Tooltip (eg: error text) 
2. if we associate supporting text in that InfoIcon, for example input's help text. For screen reader users that help text should also be associated with the Input component itself and not in an entirely different tooltip component.